sysutils/borgbackup/Portfile

```diff
@@ -23,29 +23,38 @@ long_description    BorgBackup (short: Borg) is a deduplicating backup \
 homepage            https://borgbackup.github.io/
 master_sites        pypi:b/borgbackup
 
-checksums           rmd160  cd2391b7a3d6fe1c9e096047063caea76739cd46 \
-                    sha256  fc6efd2d19454625532e0defac90cd8e8737665e10ae59edb15356ee5a99d291
+checksums           rmd160  f13b9a2428980b60c329431bb6c6a1170b1a0db2 \
+                    sha256  a1d2e474c85d3ad3d59b3f8209b5549653c88912082ea0159d27a2e80c910930 \
```

Comment:
These days we usually add a "size" field as well.

Reply:
That's right! Let me fix that